.travis.yml

```diff
@@ -24,12 +24,12 @@ env:
     - PHPUNIT_VERSION='*@dev'
 
 before_script:
-    - composer self-update
     - composer require --no-update phpunit/phpunit=$PHPUNIT_VERSION
     - composer install --prefer-dist
 
 script:
     - vendor/bin/phpunit --coverage-clover=coverage.clover
+    - if [[ $TRAVIS_PHP_VERSION == "7.0" && $PHPUNIT_VERSION == '6.*@stable' ]]; then vendor/bin/php-cs-fixer fix --diff --dry-run --verbose ; fi
```

Comment:
Coding standard must be executed in the lower PHP version supported, but can be checked in one build only to speed up the build.
I suggest to move to a `- include` signature to have a simplier variable management; I can PR a PoC for this too.